crates/bevy_winit/Cargo.toml

```diff
@@ -28,19 +28,21 @@ bevy_utils = { path = "../bevy_utils", version = "0.12.0" }
 bevy_tasks = { path = "../bevy_tasks", version = "0.12.0" }
 
 # other
-winit = { version = "0.28.7", default-features = false }
-accesskit_winit = { version = "0.15", default-features = false }
+winit = { version = "0.29", default-features = false, features = ["rwh_05"] }
```

Comment:
Documenting why this feature is needed in a comment would be nice.

Reply:
I added a comment
```
# feature rwh_05 refers to window_raw_handle@v0.5,
# updating to rwh_06 is blocked until wgpu 0.19 release lands.
```